ethereumj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -455,7 +455,6 @@ public void createContract(DataWord value, DataWord memStart, DataWord memSize)
 
         track.commit();
         getResult().addDeleteAccounts(result.getDeleteAccounts());
-        getResult().addLogInfos(result.getLogInfoList());
```

Comment:
There was merge one screen above.
Actually merge also adds deleted accounts but looks like `addDeleteAccounts` line should stay there. There's still some magic for me there.